app/models/review_request.rb

```diff
@@ -31,4 +31,11 @@ class ReviewRequest < ApplicationRecord
              foreign_key: :reviewer_id,
              inverse_of: :received_review_requests
   belongs_to :pull_request, inverse_of: :review_requests
+
+  enum status: { active: 'active', removed: 'removed' }
+  validates :status, inclusion: { in: statuses.keys }
+
+  def remove
```

Comment:
you don't need this method, rails enums already gives you some handful method, for example in your case you want to do `pr.review_requests.find_by!(reviewer: reviewer)&.removed!`